cs/cli/vowpalwabbit.h

```diff
@@ -45,7 +45,7 @@ public ref class VowpalWabbit : VowpalWabbitBase, IVowpalWabbitExamplePool
   /// </summary>
   /// <remarks>
   /// This is used to emit empty lines to cache while handling multiline examples.
-  /// Used internally by Learn(IEnumerable<String> lines)
+  /// Used internally by Learn(IEnumerable&lt;String&gt; lines)
```

Comment:
>&gt; [](start = 52, length = 4)

Is this needed for a documentation tool?

Reply:
If we want to generate documentation for the C#, then I believe so. Assuming we want to use XML docs, rather than something else.